### PR TITLE
Add color setting for setPrefix

### PR DESCRIPTION
**Pull request by madsonic**

### CODE REVIEW

**Comment by mgod** [comment]
I'm happy to include this feature, but I don't think I want to include parsing color constants as part of the library. The second version of the method seems good to me and I'd probably include something like this in the docstring:

```
You can get a color integer either using {@link android.support.v4.content.ContextCompat#getColor(android.content.Context, int)} or with {@link android.graphics.Color#parseColor(String)}. Formats copied from android.graphics.Color:
             You can use: '#RRGGBB',  '#AARRGGBB'
             or one of the following names: 'red', 'blue', 'green', 'black', 'white',
             'gray', 'cyan', 'magenta', 'yellow', 'lightgray', 'darkgray', 'grey',
             'lightgrey', 'darkgrey', 'aqua', 'fuchsia', 'lime', 'maroon', 'navy',
             'olive', 'purple', 'silver', 'teal'.
```

File: example/src/main/java/com/tokenautocomplete/TokenActivity.java [hunk 1]
```diff
@@ -63,7 +63,7 @@ protected boolean keepObject(Person person, String mask) {
 
 
         if (savedInstanceState == null) {
-            completionView.setPrefix("To: ");
+            completionView.setPrefix("To: ", "blue");
```
Comment (mgod) [inline example/src/main/java/com/tokenautocomplete/TokenActivity.java]: I think this should be:

```completionView.setPrefix("To: ", Color.parseColor("blue"));```

File: library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import android.annotation.TargetApi;
 import android.content.Context;
 import android.content.res.ColorStateList;
+import android.graphics.Color;
```
Comment (mgod) [inline library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java]: Shouldn't need this import any more.

Comment (madsonic) [inline library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java]: Done! Thanks for making this awesome library! I will be happy to help out with other parts too. Any idea when you will be planning out the next version?

Comment (mgod) [inline library/src/main/java/com/tokenautocomplete/TokenCompleteTextView.java]: I'm not sure - it depends on how quickly I get feedback on #272 and priorities on this at my job.

**Pull request merged**